sphinx_immaterial/custom_admonitions.py

```diff
@@ -413,6 +486,10 @@ def setup(app: Sphinx):
     app.connect("env-check-consistency", add_admonition_and_icon_css)
     app.connect("config-inited", on_config_inited)
 
+    # override original version directives with custom derivatives
+    for name, __ in VERSION_DIR_STYLE.items():
```

Comment:
Perhaps this should be gated on a config option?

Reply:
Well the visit/depart methods are always overridden using the `html_translator_mixin.override` decorator. The directive overrides in `setup()` would also have to be conditional as well (using the same config option).

I didn't attempt this because the original functionality is still employed. All, I'm doing here is adding CSS styling in a way that fits with our other Custom Admonitions' features.

Reply:
Done. I had to move some things into the visitor to ensure the CSS customization still works despite the value of `sphinx_immaterial_override_version_directives`.

Docs updated accordingly.